### PR TITLE
Ft embed social content 151208547

### PR DESCRIPTION
**Pull request by QUDUSKUNLE**
What does this PR do?

This PR embeds social content

Description of Task to be completed?

embed Social contents
write e2e test

How should this be manually tested?

Any background context you want to provide?

N/A

What are the relevant pivotal tracker stories?

#151208547

Screenshots (if appropriate)
screen shot 2017-10-09 at 1 06 35 pm

### CODE REVIEW

File: lib/collections/collections.js [hunk 1]
```diff
@@ -173,3 +173,8 @@ Themes.attachSchema(Schemas.Themes);
 export const Translations = new Mongo.Collection("Translations");
 
 Translations.attachSchema(Schemas.Translation);
+
+
```
Comment (mradeybee) [inline lib/collections/collections.js]: Remove extra empty line

**Review by mradeybee — APPROVED** [review]
Fix the little comment I have and you should be good to go.

File: lib/collections/collections.js (hunk 1 above)
Comment (mradeybee) [reply lib/collections/collections.js]: Also, make sure your tests pass.

**Comment by mradeybee** [comment]
Thanks @danielgindi I am merging this for @QUDUSKUNLE

**Comment by danielgindi** [comment]
@mradeybee I think you meant @Dannytebj :-)

**Pull request merged**